easybuild/easyconfigs/b/BEDTools/BEDTools-2.28.0-foss-2018b.eb

```diff
@@ -0,0 +1,39 @@
+# Author: Maxime Schmitt, University of Luxembourg
+# Author: Adam Huffman, The Francis Crick Institute
+#
+# Based on the work of: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics (SIB)
+# Biozentrum - University of Basel
+
+easyblock = 'MakeCp'
+
+name = 'BEDTools'
+version = '2.28.0'
+
+github_account = 'arq5x'
+homepage = 'https://github.com/%(github_account)s/%(namelower)s2'
+description = """The BEDTools utilities allow one to address common genomics tasks such as finding feature overlaps
+ and computing coverage. The utilities are largely based on four widely-used file formats: BED, GFF/GTF, VCF,
+ and SAM/BAM."""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
```

Comment:
This can be dropped down to `GCC-7.3.0-2.30`

Reply:
@Micket I changed this.

Reply:
Sorry, I totally missed that the dependency BamTools was at foss level back in 2018b, meaning this is also forced to foss/2018b level. Could you switch back?

Reply:
@Micket No problem, I switched it back.